tests/exts/holidays/valentines/test_bemyvalentine.py

```diff
@@ -0,0 +1,323 @@
+
```

Comment:
See my comment on another PR - https://github.com/python-discord/sir-lancebot/pull/1656#discussion_r2017828822

We don't have existing tests on this repo so I'm not sure we want these.